examples/end-to-end-example/provider.tf

```diff
@@ -2,6 +2,7 @@
 provider "ibm" {
   ibmcloud_api_key = var.ibmcloud_api_key
   region           = var.region
+  visibility       = "private"
```

Comment:
In order for CRA scan to pass, you will need to expose this as a variable in the example so that CRA can set this to `public` otherwise it fails as it has no access to the private endpoint